storage/redis.go

```
@@ -78,23 +89,28 @@ type RedisStorageConfig struct {
 
 // RedisClusterStorageManager is a storage manager that uses the redis database.
 type RedisClusterStorageManager struct {
-	db        redis.UniversalClient
+	db        *RedisManager
 	KeyPrefix string
 	HashKeys  bool
 	Config    RedisStorageConfig
 }
 
-func NewRedisClusterPool(forceReconnect bool, config RedisStorageConfig) redis.UniversalClient {
+func NewRedisClusterPool(forceReconnect bool, config *RedisStorageConfig) {
```

this should return an error

---

also let's change the name